Xamarin.Forms.Core/Registrar.cs

```diff
@@ -21,6 +21,9 @@ public class Registrar<TRegistrable> where TRegistrable : class
 
 		public void Register(Type tview, Type trender)
 		{
+			//avoid caching null renderers
```

Comment:
Yap crossed my mind that.. but decided null wasn't important .. will take your suggestion.